cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrap.scala

```diff
@@ -78,31 +79,43 @@ final class ClusterBootstrap(implicit system: ExtendedActorSystem) extends Exten
     } else if (bootstrapStep.compareAndSet(NotRunning, Initializing)) {
       log.info("Initiating bootstrap procedure using {} method...", settings.contactPointDiscovery.discoveryMethod)
 
+      ensureSelfContactPoint()
       val bootstrapProps = BootstrapCoordinator.props(discovery, joinDecider, settings)
       val bootstrap = system.systemActorOf(bootstrapProps, "bootstrapCoordinator")
       // Bootstrap already logs in several other execution points when it can't form a cluster, and why.
-      bootstrap ! BootstrapCoordinator.Protocol.InitiateBootstrapping
+      val initiateBootstrapping = selfContactPoint.map(BootstrapCoordinator.Protocol.InitiateBootstrapping)
+      initiateBootstrapping pipeTo bootstrap
```

Comment:
While I like how short this is, it might be somewhat more readable to write this something like
```
selfContactPoint.onSuccess {
  uri => bootstrap ! BootstrapCoordinator.Protocol.InitiateBootstrapping(uri)
}
```
Not a blocker though.